spotbugs/src/main/java/edu/umd/cs/findbugs/Version.java

```diff
@@ -20,28 +20,34 @@
 package edu.umd.cs.findbugs;
 
 import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
 import java.net.URL;
 import java.util.Enumeration;
 import java.util.jar.Manifest;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import javax.annotation.CheckForNull;
 
 /**
  * Version number and release date information.
  */
 public class Version {
+
+    private static final Logger LOGGER = Logger.getLogger(MethodHandles.lookup().lookupClass().getName());
+
     /**
      * SpotBugs website.
      */
     public static final String WEBSITE = "https://spotbugs.github.io/";
 
-    public final static String VERSION_STRING;
+    public static final String VERSION_STRING;
```

Comment:
I would avoid mixing unrelated changes, here and below too. even if this is due the style checks. PR's should not have side effects.

Reply:
It's to keep SonarQube happy, yeah. Unrelated, but pretty trivial, so...

Reply:
@iloveeclipse Which approach would you prefer? Second pull request on the same branch after the first one is merged? Separate Github issue just saying "cleanup for issue X"?

Reply:
Cleanup after the main fix. This way one can always do cherry picking without side effects. So yes, please next time try to not mix different changes in one commit.